### ISSUE TITLE
Face Generator Training Loss Error

### ISSUE THREAD

**Issue by Xiao-Fangff** [yhw-yhw/TalkSHOW#37]
The code is in `nets/smplx_face.py#161`

```
    def get_loss(
        self,
        pred_poses,
        gt_poses,
        pre_poses,
        aud,
        mode="training_G",
        gt_conf=None,
        exp=1,
        gt_nzero=None,
        pre_nzero=None,
    ):
        loss_dict = {}

        [b_j, b_e, b_b, b_h, b_f] = self.dim_list

        MSELoss = torch.mean(torch.abs(pred_poses[:, :, :6] - gt_poses[:, :, :6]))
        if self.expression:
            expl = torch.mean((pred_poses[:, :, -100:] - gt_poses[:, :, -100:]) ** 2)
        else:
            expl = 0

        gen_loss = expl + MSELoss

        loss_dict["MSELoss"] = MSELoss
        if self.expression:
            loss_dict["exp_loss"] = expl

        return gen_loss, loss_dict
```

The dimension of jaw is 3. So `MSELoss = torch.mean(torch.abs(pred_poses[:, :, :6] - gt_poses[:, :, :6]))` should be corrected to `MSELoss = torch.mean(torch.abs(pred_poses[:, :, :3] - gt_poses[:, :, :3]))` ?
